admin-dev/themes/new-theme/js/pages/product/components/combination-list/bulk-edition-handler.ts

```diff
@@ -64,46 +74,52 @@ export default class BulkEditionHandler {
   }
 
   private init(): void {
+    this.getBulkEditFormBtn()!.addEventListener('click', () => this.showFormModal());
+  }
+
+  private getBulkEditFormBtn(): HTMLButtonElement | null {
     const bulkEditionFormBtn = this.tabContainer.querySelector<HTMLButtonElement>(CombinationMap.bulkCombinationFormBtn);
 
     if (!(bulkEditionFormBtn instanceof HTMLButtonElement)) {
       console.error(`${CombinationMap.bulkCombinationFormBtn} was expected to be HTMLButtonElement`);
-      return;
+      return null;
     }
+
+    return bulkEditionFormBtn;
+  }
+
+  private async showFormModal(): Promise<void> {
```

Comment:
```suggestion
  private async showFormModal(bulkEditionFormBtn HTMLButtonElement): Promise<void> {
```
You could use the previously fetched button in the callback, this ay no need to add some `!` at the end of the getter to ignore TypeScript warnings/errors